api/nakadi-event-bus-api.yaml

```diff
@@ -1177,8 +1177,7 @@ definitions:
         allOf:
           - $ref: '#/definitions/EventTypeSchema'
         description: |
-          The schema for this EventType. This is expected to be a json schema in yaml format (other formats
-          might be added in the future).
```

Comment:
If you remove this then I think it will make sense to write in 'EventTypeSchema' definition that only currently supported schema type is json. What do you think?